shop/src/pages/super-admin/networks/_Form.js

```diff
@@ -129,6 +127,9 @@ const NetworkForm = ({ onSave, network, feedback, className }) => {
   if (network && !network.defaultShopConfig) {
```

Comment:
nit: maybe add a comment explaining why network.defaultShopConfig is a string but network.fallbackShopConfig is an Object :)
and maybe also TODO for us in the future regarding changing the encoding for defaultShopConfig in the back-end from string to json - I think it's pretty confusing right now :)